stub-server/src/main/java/org/zanata/rest/service/MockFileResource.java

```diff
@@ -61,6 +61,7 @@
 @Path(FileResource.SERVICE_PATH)
 public class MockFileResource implements FileResource {
     @Override
+    @Deprecated
```

Comment:
This should have an `@deprecated` tag in the javadoc pointing to an undeprecated alternative.